### PR TITLE
RavenDB-16426 Document Compression View - Explain

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: src/Raven.Studio/typescript/viewmodels/database/settings/documentsCompression.ts [hunk 1]
```diff
@@ -41,6 +43,8 @@ class documentsCompression extends viewModelBase {
         this.canAddAllCollections = ko.pureComputed(() => {
            return !!_.difference(this.allExistingCollections(), this.collectionsToCompress()).length;
         });
+        
+        this.storageReportUrl = ko.pureComputed(() => appUrl.forStatusStorageReport(this.activeDatabase().name));
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/settings/documentsCompression.ts]: please check if this doesn't break anything when you change database via top switcher. Then active database might be null for a moment?

File: src/Raven.Studio/wwwroot/App/views/database/status/storageReport.html [hunk 2]
```diff
@@ -7,53 +7,58 @@
...
+                <i data-bind="visible: $index() < $root.currentPath().length - 1" class="icon-arrow-filled-right"></i>
+            </span>
+        </div>
         <div class="flex-separator"></div>
-        <button class="btn btn-sm btn-default" data-bind="click: compactDatabase">
-            <i class="icon-compact"></i><span>Compact Database</span>
-        </button>
+        <div class="padding padding-xs margin-right margin-right-sm" title="Set collections to be compressed - Navigate to Document Compression view">
+            <small>
+                <a target="_blank" data-bind="attr: { href: documentsCompressionUrl }"><i class="icon-link margin-right margin-right-xs"></i>Document Compression</a>
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/database/status/storageReport.html]: I think it shoun't be target blank? It isn't external link, right?

File: src/Raven.Studio/typescript/viewmodels/database/settings/documentsCompression.ts (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/database/settings/documentsCompression.ts]: done

File: src/Raven.Studio/wwwroot/App/views/database/status/storageReport.html (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/database/status/storageReport.html]: done

**Review by ml054 — APPROVED** [review]

**Comment by Danielle9897** [comment]
rebased

**Review by ml054 — APPROVED** [review]

**Pull request merged**